package.json

```diff
@@ -62,9 +62,11 @@
     "@typescript-eslint/eslint-plugin": "^4.28.3",
     "@typescript-eslint/parser": "^4.28.3",
     "chai": "^4.3.4",
+    "codecov": "^3.8.3",
     "eslint": "^7.32.0",
     "eslint-config-prettier": "^8.3.0",
     "mocha": "^8.4.0",
+    "nyc": "^15.1.0",
```

Comment:
I'm pretty sure this is the only thing that is needed in the deps to collect the coverage. Not codecov. Anything I'm missing?